### PR TITLE
Hyperlink Descriptors

### PR DESCRIPTION
**Pull request by katiejiang**

### CODE REVIEW

**Comment by katiejiang** [comment]
Changes the database, so consider merging later!

File: app/main/views.py [hunk 1]
```diff
@@ -125,7 +127,7 @@ def get_associations(resource_id):
             values = set(curr)
```
Comment (HanaPearlman) [inline app/main/views.py]: The previous line should be changed to curr['value'].append(val) since now "curr" is an object not just a list!

**Comment by HanaPearlman** [comment]
One more thing-- when I add_fake_data and then look at advanced options, I can see one called "Website." So it seems like Hyperlinks are being included by accident in the advanced options, as if they were option descriptors.

**Comment by katiejiang** [comment]
ok so I implemented a really quick fix to change the color of the hyperlink but it's probably horrible style pls help me @stephanieyshi :'(

**Comment by stephanieyshi** [comment]
Can you merge in master? I'm seeing this 😢 
image

**Comment by katiejiang** [comment]
@stephanieyshi I think the issue is that you have the old maps.js code cached on your browser. Try going to history and clearing your cache, and reload the page!

**Comment by katiejiang** [comment]
screen shot 2018-03-31 at 4 12 33 pm

**Comment by HanaPearlman** [comment]
This is good to merge! Not sure where we stand on merging changes which update the db but this is great! Thanks for making the changes.

**Pull request closed without merging**